pom.xml

```diff
@@ -5,7 +5,7 @@
     <packaging>jar</packaging>
     <name>DSpace Replication Task Suite</name>
     <url>https://wiki.duraspace.org/display/DSPACE/ReplicationTaskSuite</url>
-    <version>3.3</version>
+    <version>3.x</version>
```

Comment:
Odd. This shouldn't have said 3.3.  Looks like the POM was not updated automatically by Maven after the 3.3 release:
https://github.com/DSpace/dspace-replicate/releases/tag/dspace-replicate-3.3

Could you please update this to be `3.4-SNAPSHOT`?  As 3.4 looks to be the next release we'd need to do here.